--- a/vendor/llama.cpp/CMakeLists.txt
+++ b/vendor/llama.cpp/CMakeLists.txt
@@ -583,9 +583,4 @@
     add_compile_options(-mfma -mf16c -mavx -mavx2)
 endif()

-if(LLAMA_NATIVE)
-    if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
-        add_compile_options(-march=native)
-    endif()
-endif()
-
 if (WIN32 AND CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
     add_compile_options(-pthread)
 elseif(NOT WIN32)
